controller/metrics/metrics.go

```diff
@@ -198,7 +198,11 @@ func NewMetricsServer(addr string, appLister applister.ApplicationLister, appFil
 	registry.MustRegister(resourceEventsProcessingHistogram)
 	registry.MustRegister(resourceEventsNumberGauge)
 
-	return &MetricsServer{
+	kubectlMetricsServer := kubectl.NewKubectlMetrics()
```

Comment:
Would it help, if we provide option to toggle on/off this metrics. Some might already have these metrics built in k8s and it could be duplicate.

Reply:
I could imagine a toggle being useful, but I'd like to avoid more flags if possible.

> Some might already have these metrics built in k8s

They won't have exactly _these_ metrics built-in, because only Argo CD has access to data about client-go's behavior on Argo CD components. Folks might have server-side metrics from k8s, but they won't have the client-side metrics of our clients.